inst/extdata/energy/A22.globaltech_retirement.csv

```diff
@@ -0,0 +1,38 @@
+# File: A22.globaltech_retirement.csv
+# Title: Primary energy transformation technology lifetimes and shutdown rates
```

Comment:
Any `Source:` for these files? @pkyle ?

Reply:
Do you mean A322.globaltech_retirement? Not sure why A22 would be called by this code. In either case, the capital stock survival curves are GCAM assumptions for which literature estimates are not available. The A22 assumptions were documented as thoroughly as they ever will be in Proposal 202, but it was not uploaded to Confluence so there's no link.

Reply:
`A22.globaltech_retirement.csv` was not called by this chunk. I inserted comments for all "*_retirement.csv" and added all assumption files into the new system as we discussed in #737